vue-components/stories/Button.stories.ts

```diff
@@ -10,24 +9,15 @@ export default {
 export function normal(): Component {
 	return {
 		components: { Button },
-		props: {
-			content: {
-				default: text( 'Message', 'Everything normal' ),
-			},
-			disabled: {
-				default: boolean( 'disabled', false ),
-			},
-			// naming a prop the same way the component's prop is named leads to
-			// a broken state where the property value is not passed to the component
-			// this happens with vue-cli v4.4.0 and storybook v6.0.0-rc.24
-			buttonType: {
-				default: select(
-					'Type',
-					[ 'neutral', 'primaryProgressive', 'primaryDestructive' ],
-					'neutral',
-				),
-			},
-		},
-		template: '<Button v-html="content" :disabled="disabled" :type="buttonType" />',
+		// The normal button types are all in the same story to achieve high % of visual tests coverage.
```

Comment:
Thanks for the comment! I was actually wondering why we were moving away from knobs :+1: